homeassistant/components/sonos/__init__.py

```diff
@@ -61,6 +62,7 @@
                     vol.Schema(
                         {
                             vol.Optional(CONF_ADVERTISE_ADDR): cv.string,
+                            vol.Optional(CONF_LISTENER_PORT): cv.port,
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configuration or changes.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>